### PR TITLE
DirectoryService: Add v1.0 docs for device memberOf

### PR DESCRIPTION
**Pull request by eketo-msft**

### CODE REVIEW

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/device-list-memberof.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/api-reference/v1.0/api/device-list-memberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812010114152314-3535%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Ykd%2BQZNdLQ68m0RQmn5w4FCXMyPWeWAMGQnPTYEg57I%3D&st=2018-12-01T01%3A12%3A52Z&se=2019-01-01T01%3A17%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/device-list-memberof.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/api-reference/v1.0/api/device-list-memberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812010130390753-3535%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=6QAkwQkaHeu%2FiZ691TlVi3413NeIrQddw%2BXIEoQirNI%3D&st=2018-12-01T01%3A33%3A18Z&se=2019-01-01T01%3A38%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/device-list-memberof.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/api-reference/v1.0/api/device-list-memberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812010143151006-3535%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=L3GZDs2QX5J%2BnYR8v65pAf7UvIaHR0YJI95z5GF4O4o%3D&st=2018-12-01T01%3A43%3A36Z&se=2019-01-01T01%3A48%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/v1.0/api/device-list-memberof.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+|Delegated (personal Microsoft account) | Not supported.    |
+|Application | Directory.Read.All, Directory.ReadWrite.All |
+
+## HTTP request
+<!-- { "blockType": "ignored" } -->
+```http
+GET /devices/{id | userPrincipalName}/memberOf
+```
+## Optional query parameters
+This method supports the [OData Query Parameters](https://developer.microsoft.com/graph/docs/concepts/query_parameters) to help customize the response.
```
[the remark this answers is not in the document]
Comment (eketo-msft) [reply api-reference/v1.0/api/device-list-memberof.md]: Fixed, good catch!

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/device-list-memberof.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/api-reference/v1.0/api/device-list-memberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812010153160276-3535%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=HmW3CxrX4ZJW%2BYvBrLNfK5vrj7RcOy1AvzR0nCttKk4%3D&st=2018-12-01T01%3A51%3A55Z&se=2019-01-01T01%3A56%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: concepts/changelog.md [hunk 2]
```diff
@@ -9,6 +9,14 @@ This changelog covers what's changed in Microsoft Graph, including the v1.0 and
 
 For details about known issues with Microsoft Graph APIs, see [Known issues](known-issues.md).
 
+## December 2018
```
[the remark this answers is not in the document]
Comment (eketo-msft) [reply concepts/changelog.md]: Whichever is merged second will have a merge conflict which can be resolved using the UI here.

**Comment by jthake-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/device-list-memberof.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/api-reference/v1.0/api/device-list-memberof.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/device.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/api-reference/v1.0/resources/device.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/eketo-device-memberOf/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C6%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201812062243056309-3535%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=0BBB6D7JdzBXiZ6CVrKjAiPzIZZGIRaG%2BKOrrYeGuTk%3D&st=2018-12-06T22%3A41%3A29Z&se=2019-01-06T22%3A46%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**